repairnator/docker-images/bears-checkbranches-dockerimage/README.md

```diff
@@ -0,0 +1,16 @@
+# Testing `check_branches.sh`
+
+The tests on `check_branches.sh` are in the file `check_branches_test.sh`, which relies on the testing framework [shUnit2](https://github.com/kward/shunit2).
+To run the tests, two steps are required:
+
+1. Install shUnit2:
+
+```bash
+$ sudo apt-get install shunit2
```

Comment:
Could you modify with:

```
$ sudo apt-get install shunit2 # for debian/ubuntu linux
$ brew install shunit2 # for macos
```

Reply:
Sure. I will do that after I finish my fight with the Travis build.